subjects/buzzinga/README.md

```diff
@@ -0,0 +1,70 @@
+# buzzinga
+
+### Instructions
+
+Write a function named `BuzZinga()` that takes a number as an argument, then  loop over it and prints the following:
```

Comment:
Write a function called `BuzZinga()` that takes a number as an argument, then loops over it and does the following for each iteration: